frontend/packages/app/src/clutch.config.js

```diff
@@ -4,6 +4,10 @@ module.exports = {
       trending: true,
       componentProps: {
         resolverType: "clutch.aws.ec2.v1.Instance",
+        // notes: [{
```

Comment:
I think we can just remove this for now

Reply:
I did not mean to commit this, let me remove.